src/components/Home/Home.tsx

```diff
@@ -1,35 +1,66 @@
 import React, { useState, useEffect } from 'react'
+import { fetchGetResponse } from '../../utils/apiCalls'
```

Comment:
This name is kinda unclear to me, can we change it to something more semantic?

Reply:
Suggestion? It's dynamically fetching a GET response object.